lib/extensions/conditional/active_record/base.rb

```diff
@@ -22,6 +22,14 @@ module ConditionalInstanceMethods
     def specific_conditions
       conditions.map(&:actable)
     end
+
+    # Check if all conditions are satisfied by the subject
+    #
+    # @param [Object] subject The subject that conditions are being checked on
+    # @return [Boolean] true if all conditions are met and false otherwise
+    def conditions_satisfied_by?(subject)
+      specific_conditions.all? { |condition| condition.satisfied_by?(subject) }
```

Comment:
I think the abstraction condition should implement satisfied_by? then delegate it to the actable.